docs/source/security/terminating-ssl.md

```diff
@@ -2,6 +2,8 @@
 title: Terminating SSL
 ---
 
+FIXME not super clear why we need this file rather than just linking to somebody else's docs about how to add https to an arbitrary Express app
```

Comment:
I don't think it hurts anything to keep it, and perhaps it's helpful for it to exist if only to indicate that it requires `apollo-server-express` instead of `apollo-server`?

Reply:
Fair enough